index/list.go

```diff
@@ -0,0 +1,25 @@
+package index
+
+import "github.com/kuzzleio/sdk-go/types"
+
+func (i *Index) List() (string, error) {
```

Comment:
This method should return a slice of strings, containing the list of available indexes in Kuzzle.
See the [API list method](http://docs.kuzzle.io/api-documentation/controller-index/list/) and the current [SDK listIndexes function](http://docs.kuzzle.io/sdk-reference/kuzzle/list-indexes/) documentations